play-services-core/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <permission
+        android:name="com.google.android.providers.gsf.permission.READ_GSERVICES"
+        android:label="@string/perm_gsf_read_gservices_label"
+        android:permissionGroup="android.permission-group.ACCOUNTS"
```

Comment:
This permission group doesn't seem to exist (anymore). Maybe just not put the permission in a group - original GSF also doesn't.